### ISSUE TITLE
fix examples for FX2d

### ISSUE THREAD

**Issue by kasperkamperman** [cansik/deep-vision-processing#16]
Thanks for this great library. 

Regarding the examples. In Processing 4 you need to explicitly import the FX2D library

```Please use Sketch → Import Library to add JavaFX to your sketch.```

Adding  ```import processing.javafx.*;``` solves this issue. 

However, I don't know if FX2D is really needed, just changing the size by using the default Processing mode worked fine for me too. 
```size(560, 560, FX2D);``` >>> ```size(560, 560);```

**Comment by cansik** [comment]
Yes we have to go through the examples and remove the FX2D renderer. Maybe give the examples a better folder structure would great too.
